### PR TITLE
Docker env for running melange without alpine

### PR DESCRIPTION
**Pull request by amouat**
Created a script based on apko's apko-inception for running melange.

It has a few issues currently:

 - requires a modified version of the apko-inception image with bubblewrap.
 - Should we host the image somewhere? GHCR?
 - As part of a run script, it installs a go module and makes melange. I guess the next step would be to see if this can be done with apko?
 - Currently requires "--priviliged" for bubblewrap; i'll look into the exact caps required.

Anyway, I thought I'd make it available now for people to play with, but I'm interested in what direction people think it should go in.

### CODE REVIEW

**Comment by kaniini** [comment]
(is this ready to go in?)

**Comment by amouat** [comment]
I think we should get it going using the official melange image rather than the apko dev environment, if that makes sense.

File: hack/run-devenv.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```
Comment (cpanato) [inline hack/run-devenv.sh:1]: maybe to be similar to the other scripts we have we can use

```
#!/usr/bin/env bash
```

just a comment non blocking

Comment (amouat) [reply hack/run-devenv.sh:1]: Note it gets called from within a container, so bash won't work. I can't remember if there was a problem `/usr/bin/env sh` or not.

Comment (cpanato) [reply hack/run-devenv.sh:1]: ahh so ignore my comment 🤦‍♂️

Comment (kaniini) [reply hack/run-devenv.sh:1]: `/bin/sh` is required to exist due to POSIX anyway :p

**Comment by jdolitsky** [comment]
Using this script, the build fails with:
```
2022/07/01 00:14:59 melange (hello/s390x): bwrap: Creating new namespace failed: Invalid argument
Error: failed to build package: unable to run pipeline: exit status 1
```

**Comment by jdolitsky** [comment]
Going to close this one in relation to #76

**Pull request closed without merging**